apis/api-journeys/src/workers/migrations/button-blocks/update-submit-enabled.ts

```
@@ -0,0 +1,42 @@
+import { PrismaClient } from '.prisma/api-journeys-client'
```

ESLint

[error] Useless path segments for ".prisma/api-journeys-client", should be "./.prisma/api-journeys-client"

(import/no-useless-path-segments)